docs/wiki/feature/refuel.md

```diff
@@ -19,44 +19,41 @@ This adds the option to refuel vehicles. The basic concept of refuel a vehicle i
 ## 2. Usage
 
 ### 2.1 Start refueling vehicles
-- Interact with the fuel truck <kbd>⊞&nbsp;Win</kbd> (ACE3 default key bind `Interact Key`).
+- Interact with the fuel source <kbd>⊞&nbsp;Win</kbd> (ACE3 default key bind `Interact Key`).
 - Select `Take fuel nozzle`. Your character will holster his weapon and receive a fuel nozzle.
-- Walk to the vehicle you want to refuel and interact with it.
-- Select `Connect fuel nozzle`.
-- Place the nozzle on the vehicle and interact with it.
-- Select `Start fueling`. The vehicle is now being refuel.
+- Walk close to the vehicle you want to refuel and follow the instructions on the screen to connect the nozzle.
+- Refueling starts automatically after nozzle connecting.
```

Comment:
Only if engine is off and other conditions are satisfied IIRC from the PR, you can still start (and stop) it manually if it doesn't auto-start.